Components/DB/Dbal/DbalStepsProvider.php

```diff
@@ -278,10 +278,17 @@ protected function performQuery(array $configuration, &$context, array $paramete
 
     /**
      * @param $context
+     *
      * @return object
      */
     protected function getConnection($context)
     {
-        return $this->doctrine->getConnection();
+        $connectionName = null;
+
+        if (!empty($context['options']['db_connection_name'])) {
```

Comment:
I am not sure that with that name we are talking about a connection name or a database name...
Because in the Platform Bundle, when we use 
$databaseName = $context['options']['db_connection_name'];
it means that db_connection_name is a database name.

Reply:
I am afraid there will be confusion between database and connection, and the way they are setup in databases.yml